### ISSUE TITLE
Accessing savestates page hangs Amiberry

### ISSUE THREAD

**Issue by wn2000** [midwan/amiberry#735]
**To Reproduce**
After having created a savestate, go back to the savestates page, select the slot where the savestate is in, the savestate screen is shown, now try to move to another slot, at this moment it hangs. In the console where Amiberry is launched, it shows:
```
Assertion failure at SDL_DestroyTexture_REAL (.../sdl2-2.0.7/src/render/SDL_render.c:2038), triggered 1 time:
  'texture && texture->magic == &texture_magic'

Abort/Break/Retry/Ignore/AlwaysIgnore? [abriA] :
```
If I type `A` (AlwaysIgnore) I can recover the control and can actually load the savestate and go back to the game. But clearly there's a bug somewhere. If the user do not have access to the command window the process has to be killed to recover.

**Additional Info**
- Version: 3.3 built from source
- Platform: Generic aarch64 on RK3328

**Comment by midwan** [comment]
I cannot recreate this in the latest `dev` version, so I will assume it's fixed there. :)

![Amiberry - Amiberry_2020-12-28_18-32-48](https://user-images.githubusercontent.com/3877365/103232664-57cec980-493b-11eb-9ea4-f1fcd98a382f.png)

**Comment by wn2000** [comment]
Thanks I'll try the dev build.
On a separate note, I built SDL 2.0.8 from source and the issue also disappeared.
I remember seeing an assertion setting when building SDL2. My guess is that the default assertion setting in SDL 2.0.8 might be to ignore any failure, and somehow the built-in SDL 2.0.7 on my system asks for a response. My initial plan was to modify SDL code to print out a backtrace at assertion failure, but since the issue does not appear again with the custom built SDL 2.0.8, I did not dig further.
